stable/prometheus/templates/tests/test-server-config.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-conf-{{ template "prometheus.name" . }}-{{ .Values.server.name }}
+  labels:
+    # TODO: decide on labels
```

Comment:
see https://github.com/helm/charts/pull/9555

items to take into account:
1. don't overlap with SVC label selector
2. allow ease to cleanup the test pods afterwards (see description https://github.com/helm/charts/pull/9585#issue-234045534)